identity-account/src/error.rs

```diff
@@ -18,7 +18,7 @@ pub enum Error {
   /// Caused by errors from the [identity_credential] crate.
   #[error(transparent)]
   CredentialError(#[from] identity_credential::Error),
-  /// Caused by errors from the [crypto] crate.
+  /// Caused by errors from the `crypto` crate.
   #[error(transparent)]
   AccountCoreError(#[from] identity_account_storage::Error),
```

Comment:
The description is incorrect, and shouldn't this be `AccountStorageError`? Latter doesn't matter though.

```suggestion
  /// Caused by errors from the [identity_account_storage] crate.
  #[error(transparent)]
  AccountCoreError(#[from] identity_account_storage::Error),
```